server/utils/modeValue.js

```diff
@@ -0,0 +1,6 @@
+const mode = words => (
+  words.sort((a, b) => words.filter(v => v === a).length
+  - words.filter(v => v === b).length).pop()
+);
+
+export default mode;
```

Comment:
Nice work on writing this utility method using functional Javascript style. I think you can make readers of your code understand what the utility does by using  JSDoc comments and also using more descriptive variable names for the variables in your array methods.

Reply:
Good one. Duly noted. I'll make the changes now